make/StaticLibsImage.gmk

```diff
@@ -49,6 +49,13 @@ $(foreach m, $(ALL_MODULES), \
   $(eval TARGETS += $$(COPY_STATIC_LIBS_$m)) \
 )
 
+$(eval $(call SetupCopyFiles, COPY_STATIC_LIBS_libjvm, \
+    SRC := $(HOTSPOT_OUTPUTDIR)/variant-$(JVM_VARIANT_MAIN)/libjvm/objs/static, \
+    DEST := $(STATIC_LIBS_IMAGE_DIR)/lib, \
+    FILES := $(filter %$(STATIC_LIBRARY_SUFFIX), \
+        $(call FindFiles, $(HOTSPOT_OUTPUTDIR)/variant-$(JVM_VARIANT_MAIN)/libjvm/objs/static))))
```

Comment:
I would recommend using `wildcard` instead of `FindFiles` as the files are expected to be in a single directory (no recursive directory searching needed). We also try to put the closing double braces on a new line matching the initial eval in indentation.

```suggestion
    FILES := $(wildcard $(HOTSPOT_OUTPUTDIR)/variant-$(JVM_VARIANT_MAIN)/libjvm/objs/static/*$(STATIC_LIBRARY_SUFFIX)), \
))
```